pennylane_lightning/lightning_tensor/_tensornet.py

```diff
@@ -177,7 +179,9 @@ def __init__(
         elif self._method == "tn":
             self._tensornet = self._tensornet_dtype()(self._num_wires)
         else:
-            raise DeviceError(f"The method {self._method} is not supported.")
+            raise DeviceError(
+                f"The method {self._method} is not supported. Only supported methods are mps or tn."
```

Comment:
Let's revert this line here to ensure this PR is only for docs. Changes in the code might break the unit tests. CC: @maliasadi

Reply:
I think you're checking an old comment! I reverted this change. If not, let's do so :100:

Reply:
Oh, no! I didn't submit my suggestions -- thanks for fixing this :1st_place_medal: